e2e/test/scenarios/dashboard-cards/click-behavior.cy.spec.js

```diff
@@ -195,6 +195,15 @@ describe("scenarios > dashboard > dashboard cards > click behavior", () => {
       editDashboard();
 
       getDashboardCard().realHover().icon("click").click();
+
+      // When the default menu is selected, it should've visual cue (metabase#34848)
+      cy.get("aside")
+        .findByText("Open the Metabase drill-through menu")
+        .parent()
+        .parent()
+        .should("have.attr", "aria-selected", "true")
+        .should("have.css", "background-color", "rgb(80, 158, 227)");
```

Comment:
I added the aria-selected attribute and included it in the test. I've also tested whether the menu has the expected background-color for the selected state. Do you have any input on this matter? @kamilmielnik